bazel/foreign_cc/BUILD

```diff
@@ -52,6 +52,7 @@ envoy_cmake_external(
         "CARES_SHARED": "no",
         "CARES_STATIC": "on",
         "CMAKE_BUILD_TYPE": "RelWithDebInfo",
+        "CMAKE_RANLIB": "",
```

Comment:
Can you maybe add a comment on why we are doing this here and below? (OK to refer below comments back to here if it's long).

Reply:
Added one, lmk if you don't think it gets the point across well enough